pkg/test/utils_test.go

```diff
@@ -40,30 +42,30 @@ const (
 	dataFormGalleries    = "fixtures/crud/form_galleries.json"
 )
 
-// check if we are working with the test db
-func checkIsTestDB() bool {
-	s := strings.Split(os.Getenv("MONGODB_URL"), "_")
-	return (s[len(s)-1] == "test")
-}
+var (
+	envmongodburl string
+	testDBname    string
+)
 
 // remove the test database
-func emptydb() {
-	if test := checkIsTestDB(); !test {
-		log.Fatalf("Fail to setup test database with %s", os.Getenv("MONGODB_URL"))
-	}
-	deb := db.NewMongoDB(os.Getenv("MONGODB_URL"))
+func emptyDB() {
+	deb := db.NewMongoDB(testDBname)
 	defer deb.Close()
 
 	if e := deb.DB.DropDatabase(); e != nil {
 		log.Fatalf("Fail removing DropDatabase %s. Error: %v", deb.DB.Name, e)
 	}
 }
 
+func recoverEnvVariables() {
```

Comment:
This is an _excellent_ idea!  Love how complete the thought becomes when you return the db to the env.